OutOfSchool/OutOfSchool.WebApi/Extensions/ElasticsearchMappingExtensions.cs

```diff
@@ -20,7 +20,9 @@ public static WorkshopES ToESModel(this WorkshopDTO workshopDto)
                         dest => dest.Keywords,
                         opt =>
                             opt.MapFrom(src =>
-                                string.Join('¤', src.Keywords.Distinct())));
+                                string.Join('¤', src.Keywords.Distinct())))
```

Comment:
i totally forgot about this file :) can you add a `TODO` somewhere that some tine later we need to create a mapping profile instead of extensions.

Reply:
Ok ;)